Assets/_BForBoss/_Character/Scenes/Sandbox.unity

```diff
@@ -2740,10 +2729,10 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   _playerBehaviour: {fileID: 2217804276623157996}
+  _actionAsset: {fileID: -944628639613478452, guid: 11b5fd57f5ed74ef0a6fcf19057db823, type: 3}
```

Comment:
The BaseWorldManager now needs the `InputActionAsset` to handle the input mapping between PlayState and UI State.

![image](https://user-images.githubusercontent.com/5779541/187999002-dc195c82-2685-4f32-a07e-0a6c8f5ee6aa.png)